web/src/features/map/Map.tsx

```diff
@@ -232,6 +235,10 @@ export default function MapPage(): ReactElement {
   };
 
   const onLoad = () => {
+    if (data?.callerLocation) {
+      const map = mapReference.current?.getMap();
+      map?.flyTo({ center: [data?.callerLocation[0], data?.callerLocation[1]] });
```

Comment:
Nitpick, as the if-check above should ensure that we have something here :)
```suggestion
      map?.flyTo({ center: [data.callerLocation[0], data.callerLocation[1]] });
```